.github/workflows/codeql-config.yml

```
@@ -0,0 +1,4 @@
+paths-ignore:
```

actionlint (1.7.12)

[error] "jobs" section is missing in workflow

(syntax-check)